sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -10,6 +10,8 @@ public partial interface ITableEntity
     public partial class TableClient
     {
         protected TableClient() { }
+        public TableClient(string tableName, string connectionString) { }
```

Comment:
We have different ordering of parameters in storage: https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.specialized.blockblobclient.-ctor?view=azure-dotnet#Azure_Storage_Blobs_Specialized_BlockBlobClient__ctor_System_String_System_String_System_String_

and EH https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.eventhubs.producer.eventhubproducerclient.-ctor?view=azure-dotnet#Azure_Messaging_EventHubs_Producer_EventHubProducerClient__ctor_System_String_System_String_

Should we go with connectionString/endpoint first?

Reply:
The main difference is that `tableName` is always required.

Reply:
But connectionString/endpoint are always required as well

Reply:
fair - I'll move it.